.github/workflows/ci.yml

```diff
@@ -159,10 +164,15 @@ jobs:
           sudo apt-get -y update
           sudo apt-get -y install protobuf-compiler libprotobuf-dev
 
+      - name: Setup FoundationDB
+        uses: foundationdb-rs/foundationdb-actions-install@v2.0.0
+        with:
+          version: "7.1.30"
+
       - name: Run cargo test
         run: |
-          cargo build --locked --no-default-features --features storage-mem
-          (&>/dev/null ./target/debug/surreal start --log trace --user root --pass root memory &)
+          cargo build --locked --no-default-features --features storage-fdb
+          (&>/dev/null ./target/debug/surreal start --log trace --user root --pass root fdb:/etc/foundationdb/fdb.cluster &)
```

Comment:
@rushmorem why are we using `fdb` as the backing test database here?

Reply:
@tobiemh I added the motive behind this in the PR description:

> Both the memory engine and the foundationdb engine have similar compile times because FoundationDB uses a pre-compiled shared library. I think it's better to run remote engine integration tests against a persistent engine which is actually meant for production use.